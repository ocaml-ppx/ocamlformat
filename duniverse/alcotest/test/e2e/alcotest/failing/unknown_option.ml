let () =
  Alcotest.run __FILE__
    [ ("alpha", [ Alcotest.test_case "1" `Quick (fun () -> ()) ]) ]
