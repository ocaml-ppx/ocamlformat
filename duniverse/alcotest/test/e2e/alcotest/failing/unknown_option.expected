unknown_option.exe: unknown option `--dry-runn'.
Usage: unknown_option.exe COMMAND ...
Try `unknown_option.exe --help' for more information.
